protocol/avdl/keybase1/kbfs_common.avdl

```diff
@@ -51,4 +51,32 @@ protocol kbfsCommon {
     Folder folder;
     int requestID;
   }
+
+  record FSSyncStatusRequest {
+    int requestID;
+  }
+
+  record FSPathSyncStatus {
+    boolean publicTopLevelFolder;
```

Comment:
Agreed it's awkward but I'm not going to change it as part of this PR since we use that convention in lots of places.  I don't understand your `isTLF` suggestion, but an enum with `Public` or `Private` would probably be better in the future.